Cargo.toml

```diff
@@ -14,8 +14,9 @@ bson = "0.14"
 serde_json = "1.0"
 serde = "1.0"
 libc = "0.2"
-jsonpath_lib = { git="https://github.com/RedisJSON/jsonpath.git", branch="generic_json_path" }
-redis-module = { version="0.22", features = ["experimental-api"]}
+jsonpath_lib = { git = "https://github.com/RedisJSON/jsonpath.git", branch = "generic_json_path" }
+#redis-module = { version="0.22", features = ["experimental-api"]}
+redis-module = { git = "https://github.com/RedisLabsModules/redismodule-rs.git", branch = "omer-short-read", features = ["experimental-api"] }
```

Comment:
Make sure to change that before merge ...

Reply:
Sure, we need to bump redis-module version and use the new one here.